File: Python/day10.py
# n = 14
# count = 0
# while n != 0:
#     n = (n & (n << 1))
#
#     count += 1
#     print("N:", n)
#     print("count:", count)
#
# print(count)

n = 34829
bn = "{0:b}".format(n)
print(len(max(bn.split('0'))))

File: Ruby/day10.rb
n = 65535
aux = n.to_s(2).split(/0+/)

result = aux.reduce do |acc, elem|
  acc.size > elem.size ? acc : elem
end


puts n.to_s(2)
puts 'split:'
puts aux

sleep(3)
puts 'result:'
puts result
puts 'count:'
puts result.size
